ORION/scripts/test-prow-step.sh

```
@@ -0,0 +1,162 @@
+export SHARED_DIR="${SHARED_DIR:-/tmp/prow-shared}"
+
+mkdir -p "$ARTIFACT_DIR"
+mkdir -p "$SHARED_DIR"
+
+# ── Mock Prow Secrets ──────────────────────────────────────────────────────────────
+SECRET_DIR="/tmp/prow-secret-perfscale-prod"
+mkdir -p "$SECRET_DIR"
+
+if [[ -z "${ES_SERVER:-}" ]]; then
+    echo "ES connection setup:"
+    read -p "ES Host (e.g., myhost:9200): " ES_HOST
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)